packages/api/prisma/schema.prisma

```diff
@@ -14,22 +14,27 @@ datasource db {
   extensions = [pgcrypto]
 }
 
+enum EmailStatus {
+  UNVERIFIED
+  VERIFIED
+  BOUNCED
+  COMPLAINED
+}
+
 /// A user who can authenticate into the system.
 model AuthUser {
   /// The ID of the user.
-  id            String           @id @default(dbgenerated("generate_ulid()")) @db.Uuid
+  id           String           @id @default(dbgenerated("generate_ulid()")) @db.Uuid
   /// The display name of the user.
-  name          String?
-  /// The email address for logging in and notifications.
-  email         String?          @unique
```

Comment:
I removed the email from the user since it is now stored on the primary auth key.